tmt-app/src/Components/BobaBot.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import BobaBotLarge from "../Images/art/boba_bot_Large.svg";
+import { Box, Typography } from "@mui/material";
+
+function BobaBot({ text }) {
+  return (
+    <Box
+      sx={{
+        alignItems: "center",
+        display: "inline-flex",
+        position: "relative",
+      }}
+    >
+      <img src={BobaBotLarge} alt={"Mascot Icon"} />
```

Comment:
Alt messages can and should be more descriptive. In this case I would add the prop text to the alt. Something like "mascot icon with speech bubble that says..."